css/style.css

```diff
@@ -0,0 +1,356 @@
+  font-weight: 700;
+  font-size: 20px;
+  width: auto;
+  height: auto;
+  color: #fff;
+  text-decoration: none;
+  margin-left: 1em;
+}
+
+#navbar {
+  margin-left: auto;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+#navbar > a {
+  display: block;
+  width: 70px;
+  text-decoration: none;
+  height: auto;
+  color: white;
+  margin: 3px;
+}
+
+#home-content {
+  display: flex;
+  width: 100%;
+  height: 100vh;
+  background: #1c1a19;
+}
+
+.left-content {
+  width: 60px;
+  height: 100vh;
+  color: white;
+  margin: 10px 25px 5px 5px;
+  padding: 10px 20px 20px 5px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.desktop-version-list {
+  list-style-type: none;
+  align-self: center;
+}
+
+.desktop-version-list > li {
+  margin: 2px;
+  padding: 0;
+}
+
+.main-content {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: start;
+  text-align: left;
+  width: 100%;
+  min-height: 100vh;
+  margin: 5px 10px 15px 10px;
+  padding: 40px 10px 120px 15px;
+  flex: 1 1 100%;
+}
+
+.name-text {
+  color: #f55800;
+  font-family: 'Crete Round', sans-serif;
+  text-align: left;
+  font-style: normal;
+  font-weight: 400, regular;
+  font-size: 72px;
+  line-height: 82px;
+  margin: 5px 10px 5px 5px;
+  align-self: start;
+}
+
+.job-title {
+  color: #fff;
+  font-family: "Inter", sans-serif;
+  text-align: left;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 32px;
+  line-height: 44px;
+  margin: 5px 10px 5px 5px;
+}
+
+.job-description {
+  color: #fff;
+  text-align: left;
+  font-family: "Inter", sans-serif;
+  font-weight: 400, regular;
+  align-self: start;
+  font-style: normal;
+  font-size: 20px;
+  line-height: 32px;
+  margin-left: 5px;
+}
+
+#skill-text > ul {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding-left: 5px;
+}
+
+.img-icons {
+  width: 25px;
+  height: 25px;
+  margin: 7px;
+  background: white;
+  border: 0;
+}
+
+#social-icons {
+  display: none;
+}
+
+.right-content {
+  width: 100%;
+  height: 100vh;
+  transform: rotate(-90deg);
+  -ms-transform: rotate(-180deg);
+  -webkit-transform: rotate(-2deg);
+  margin: 0;
+  background-image: url("../images/Illustration-headerpic.png");
+  background-repeat: no-repeat;
+  background-position: bottom right;
+  background-size: auto 600px;
+}
+
+.mobile-image {
+  display: none;
+}
+
+.btn {
+  color: #fff;
+  font-size: 1.5rem;
+  border: 0;
+  background: 0;
+}
+
+@media All And (max-width: 500px) {
+  body {
+    margin: 0;
+    padding: 0;
+  }
+
+  .container {
+    display: flex;
+    flex-direction: column;
+    margin: auto;
+    height: auto;
+    color: #fff;
+  }
+
+  .site-header {
+    overflow: hidden;
+    background: #333;
+    position: relative;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  #title-text > h3 {
+    margin: 2px;
+    padding: 35px;
```

Comment:
As suggested by the previous reviewer, try removing some padding to your header to make it look closer to the Figma template :wink:

Reply:
Done as advised